src/chrome/extension/scripts/background.ts

```diff
@@ -146,6 +146,7 @@ core = new CoreConnector(browserConnector);
 var oAuth = new ChromeTabAuth();
 browserConnector.onUpdate(uproxy_core_api.Update.GET_CREDENTIALS,
                          oAuth.login.bind(oAuth));
+var ui = new user_interface.UserInterface(core, browserApi);
```

Comment:
@trevj I realized we could save a layer of message-passing indirection by just using our own UserInterface instance here.